.projenrc.js

```diff
@@ -44,6 +43,7 @@ const project = new typescript.TypeScriptProject({
     "eslint-plugin-import",
     "mockpackage@file:test/mockpackage",
   ] /* Build dependencies for this module. */,
+  peerDeps: ["typescript"],
```

Comment:
Is this a breaking change? Do we need to update projen?

Reply:
no? projen should come with typescript. plus, `functionless` consumes the closure serializer, so this will be subsumed by `functionless`.

Reply:
(From a test-project)

![image](https://user-images.githubusercontent.com/289213/174958103-a5feb1c0-a274-4fe0-9946-93ff90797cbd.png)